src/Components/Form/Form.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+import './Form.css';
+
+interface Props {
```

Comment:
I think I've seen the convention for naming these props be something like `FormProps` just so that the name is tied to the component (same for state).